client/src/Components/Appointments/AppointmentFormModal.js

```diff
@@ -142,6 +145,86 @@ export default function AppointmentFormModal(props) {
     });
   };
 
+  // Appointment validation function
+  const handleSubmitModal = (e, appointment, patient) => {
```

Comment:
It's up to you to decide which method you want to use.  Please choose one and discard other.  It's not ideal to keep unused lines of code.